pkgs/data/fonts/blobmoji/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cairo, pngquant, zopfli
+, which, pythonPackages, pkgconfig, imagemagick }: stdenv.mkDerivation rec {
+  pname = "blobmoji";
+  version = "2019-06-14-Emoji-12";
+
+  src = fetchFromGitHub {
+    owner = "C1710";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "19xwjarpn40jyibp7d2g0ss2c9hifrskkig234wdng08gc1p4z84";
+  };
+
+  buildInputs = [ cairo ];
+  nativeBuildInputs = [ cairo pngquant zopfli which pkgconfig imagemagick ]
+                      ++ (with pythonPackages; [ python fonttools nototools ]);
+  enableParallelBuilding = true;
+
+  postPatch = ''
+    sed -i 's,^PNGQUANT :=.*,PNGQUANT := ${pngquant}/bin/pngquant,' Makefile
```

Comment:
You should be able to use `makeFlags = [ "PNGQUANT=pngquant" ];` instead of patching the file. Hopefully, they will default to this in the future [like upstream Noto did](https://github.com/googlefonts/noto-emoji/blob/833a43d03246a9325e748a2d783006454d76ff66/Makefile#L21).